README.md

```diff
@@ -2,23 +2,26 @@
 
 [![Slack Status](http://slack.openpolicyagent.org/badge.svg)](https://slack.openpolicyagent.org) [![Build Status](https://github.com/open-policy-agent/opa/workflows/Post%20Merge/badge.svg?branch=main)](https://github.com/open-policy-agent/opa/actions) [![Go Report Card](https://goreportcard.com/badge/open-policy-agent/opa)](https://goreportcard.com/report/open-policy-agent/opa) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1768/badge)](https://bestpractices.coreinfrastructure.org/projects/1768) [![Netlify Status](https://api.netlify.com/api/v1/badges/4a0a092a-8741-4826-a28f-826d4a576cab/deploy-status)](https://app.netlify.com/sites/openpolicyagent/deploys)
 
-The Open Policy Agent (OPA) is an open source, general-purpose policy engine that enables unified, context-aware policy enforcement across the entire stack.
+The Open Policy Agent (OPA) project is an open source, general-purpose policy engine that enables unified, context-aware policy enforcement across the entire stack.
 
-OPA is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) as a graduated project. If you are an organization that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details read the CNCF [announcement](https://www.cncf.io/announcements/2021/02/04/cloud-native-computing-foundation-announces-open-policy-agent-graduation/).
+OPA is proud to be a graduated project in the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) landscape. For details read the CNCF [announcement](https://www.cncf.io/announcements/2021/02/04/cloud-native-computing-foundation-announces-open-policy-agent-graduation/).
 
 ## Want to learn more about OPA?
 
-- See [openpolicyagent.org](https://www.openpolicyagent.org) to get started with documentation and tutorials.
-- See [blog.openpolicyagent.org](https://blog.openpolicyagent.org) for blog posts about OPA and policy.
-- See [ADOPTERS.md](./ADOPTERS.md) for a list of production OPA adopters and use cases.
-- See [the Roadmap slides](https://docs.google.com/presentation/d/16QV6gvLDOV3I0_guPC3_19g6jHkEg3X9xqMYgtoCKrs/edit?usp=sharing) for a snapshot of high-level OPA features in-progress and planned.
-- Try [play.openpolicyagent.org](https://play.openpolicyagent.org) to experiment with OPA policies.
-- Join the conversation on [Slack](https://slack.openpolicyagent.org).
+- Go to [openpolicyagent.org](https://www.openpolicyagent.org) to see an overview of the OPA and its resources.
+- Read [docs.openpolicyagent.org](https://www.openpolicyagent.org/docs/latest/) to get started with documentation and tutorials.
+- Browse [blog.openpolicyagent.org](https://blog.openpolicyagent.org) for news about OPA, community, policy and authorization.
+- Check out [ADOPTERS.md](./ADOPTERS.md) for a list of production OPA adopters. 
+    - Submit a PR! We'd love to add your company to the list!
```

Comment:
I think "organization" is better than "company", as I'd love to see any type of orgs in that list, however unlikely :)

Reply:
I'm tempted to pull these two bullets into one or two sentences at the top of the file as a simple call to action for people that are visiting the repo. Something like:

```
Check out the ADOPTERS.md file for a list of production adopters. Does your 
organization use OPA in production? Support the OPA project by submitting a 
PR to add your organization to the list with a short description of your OPA use cases!
```